common/app/model/content.scala

```diff
@@ -472,14 +472,20 @@ object Article {
       "videoDuration" -> videoDuration
     ) ++ bookReviewIsbn ++ AtomProperties(content.atoms)
 
+    val author = if (tags.contributors.nonEmpty) {
+      tags.contributors.map(_.metadata.webUrl).mkString(",")
+    } else {
+      content.trail.byline.getOrElse("Guardian Staff")
```

Comment:
is guardian staff a safe fallback? Maybe we should fall back to an empty string where there is no byline?

Reply:
I am not entirely sure on this to be honest. I based the fallback off: https://github.com/guardian/frontend/blob/master/common/app/common/TrailsToRss.scala#L111 which uses the same approach.

The difficulty is that the whole issue I'm trying to fix here is to prevent author being empty (as it's considered invalid).